tour_model_eval/v-measurel_cutoff_bins_single_user.ipynb

```diff
@@ -216,10 +230,55 @@
     "labels_pred"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "first-campbell",
+   "metadata": {},
+   "source": [
+    "Note: the trips order in labels_true and labels_pred should be the same. Using timestamp to compare the trips in bin_trips and those in bins"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "unlimited-waste",
+   "id": "dental-universal",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bin_trips_ts = pd.DataFrame(data=[i[\"data\"][\"start_ts\"]for i in bin_trips])\n",
```

Comment:
Readability improvement: `i` is generally an index (e.g. 2) `trip` is a trip object that you can get data from

```suggestion
    "bin_trips_ts = pd.DataFrame(data=[trip[\"data\"][\"start_ts\"]for trip in bin_trips])\n",
```